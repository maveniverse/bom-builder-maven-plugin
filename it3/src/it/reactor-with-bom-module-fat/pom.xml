<?xml version="1.0" encoding="UTF-8"?>
<!--
  ~ Licensed under the Apache License, Version 2.0 (the "License");
  ~ you may not use this file except in compliance with the License.
  ~ You may obtain a copy of the License at
  ~
  ~ http://www.apache.org/licenses/LICENSE-2.0
  ~
  ~ Unless required by applicable law or agreed to in writing, software
  ~ distributed under the License is distributed on an "AS IS" BASIS,
  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  -->
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
  <modelVersion>4.0.0</modelVersion>

  <parent>
    <groupId>org.apache</groupId>
    <artifactId>apache</artifactId>
    <version>33</version>
  </parent>

  <groupId>reactor-with-bom-module</groupId>
  <artifactId>reactor-with-bom-module</artifactId>
  <version>1.0-SNAPSHOT</version>
  <packaging>pom</packaging>

  <name>Test to create reactor bom</name>

  <modules>
    <module>api</module>
    <module>impl</module>
    <module>bom</module>
  </modules>

  <properties>
    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    <version.slf4j>1.7.36</version.slf4j>
  </properties>
</project>
