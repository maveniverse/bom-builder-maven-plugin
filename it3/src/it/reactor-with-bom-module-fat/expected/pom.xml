<?xml version="1.0" encoding="UTF-8"?>
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>reactor-with-bom-module-fat</groupId>
  <artifactId>bom</artifactId>
  <version>1.0-SNAPSHOT</version>
  <packaging>pom</packaging>
  <name>Test to create reactor bom</name>
  <description>Test to create reactor bom description</description>
  <url>https://maveniverse.eu/bom/</url>
  <licenses>
    <license>
      <name>Apache-2.0</name>
      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
  <scm>
    <connection>scm:git:git@github.com:maveniverse/bom-builder-maven-plugin.git</connection>
    <developerConnection>scm:git:git@github.com:maveniverse/bom-builder-maven-plugin.git</developerConnection>
    <url>https://github.com/maveniverse/bom-builder-maven-plugin</url>
  </scm>
  <properties>
    <version.org.slf4j>1.7.36</version.org.slf4j>
    <version.reactor-with-bom-module-fat>1.0-SNAPSHOT</version.reactor-with-bom-module-fat>
  </properties>
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>org.slf4j</groupId>
        <artifactId>slf4j-api</artifactId>
        <version>${version.org.slf4j}</version>
      </dependency>
      <dependency>
        <groupId>org.slf4j</groupId>
        <artifactId>slf4j-simple</artifactId>
        <version>${version.org.slf4j}</version>
      </dependency>
      <dependency>
        <groupId>reactor-with-bom-module-fat</groupId>
        <artifactId>api</artifactId>
        <version>${version.reactor-with-bom-module-fat}</version>
      </dependency>
      <dependency>
        <groupId>reactor-with-bom-module-fat</groupId>
        <artifactId>impl</artifactId>
        <version>${version.reactor-with-bom-module-fat}</version>
      </dependency>
    </dependencies>
  </dependencyManagement>
</project>
